### ISSUE TITLE
Mindbreaker illusions float

### ISSUE THREAD

**Issue by ghost** [HippieStationArchive/HippieStation13-1#2800]
Mindbreaker illusion people are a pixel or two higher on the y axis than they're supposed to be. It makes it even more obvious that they're not real.

**Comment by GenericGoose** [comment]
just the mindbreaker? pretty sure it should be same for all illusions, so source doesn't matter

**Comment by ghost** [comment]
I've only seen it with mindbreaker, since illusions aren't all that common.

**Comment by GenericGoose** [comment]
SM shard and lings can do hallucinations too
anyway, source shouldn't matter
huillicination code seems messy 
i think tg made it better and added some system for replacing sprites client-side

**Comment by ghost** [comment]
Ling hallucinations are just mindbreaker my man.

But yeah, I doubt it does I just haven't tested radiation hallucinations.

Point being sprites are fucked ree.

**Comment by ArcLumin** [comment]
yeah, all illusions are the same code. They're just images on the client machine that float around
